_button.scss

```diff
@@ -0,0 +1,71 @@
+// button
+// ===
+
+// Configurable Variables
+// ---
+
+//Button Styles
+//---
+
+$button__padding: $vertical-input-padding $horizontal-input-padding !default;
+
+$button__border: if(variable-exists(border), $border, 1px solid $neutral-50) !default;
+$button__border-radius: if(variable-exists(border-radius), $border-radius, 0) !default;
+
+
+$button__font-color: inherit !default;
+$button__font-family: inherit !default;
+$button__font-size: inherit !default;
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+
+
+// Button root
+// ---
+
+.c-button {
+    min-height: $tap-size;
+    display:inline-block;
+    margin: 0;
+    cursor:pointer;
+    background: none;
+
+    text-align:center;
+    border:none;
+    line-height: normal;
+    white-space: normal;
+
+    span {
```

Comment:
Also, this selector should certainly come after the `user-select` and `-webkit-appearance` properties. I would argue that it should really be brought out into its own block as `.c-select > span` to emphasize that it’s a sub-component.